sshd/src/test/scala/ammonite/repl/sshd/SshServerTests.scala

```diff
@@ -0,0 +1,89 @@
+package ammonite.repl.sshd
+
+import ammonite.ops.Path
+import com.jcraft.jsch.{JSchException, Session}
+import org.scalacheck.Prop.forAll
+import utest._
+import utest.framework.TestSuite
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, Promise}
+import scala.language.postfixOps
+
+object SshServerTests extends TestSuite with ScalaCheckSupport with SshTestingTraits {
+  override val tests = TestSuite {
+    'canConnectAndAuthenticate{
+      withTmpDirectory { implicit tmpDir ⇒
```

Comment:
The reasoning behind this is to have one directory used for all runs performed by ScalaCheck. Tests will generate big IO creating & removing files for every ScalaCheck run, and I don't currently see difference between having dedicated directory for every run and having one directory to run all tests.